docs/data/migration/migration-pickers-v5/migration-pickers-v5.md

```diff
@@ -101,13 +154,101 @@ const theme = createTheme({
 });
 ```
 
+### Remove the keyboard view
```

Comment:
If we want to name the sections with stuff people have to actually do, then I don't know how to name this one